internal/handlers/auth_handler.go

```
@@ -0,0 +1,178 @@
+)
+
+type RegistrationRequest struct {
+	Name                 string `json:"name" binding:"required,min=2"`
+	FamilyName          string `json:"family_name" binding:"required,min=2"`
+	Phone               string `json:"phone" binding:"required" example:"+38 (012) 345-67-89"`
+	Email               string `json:"email" binding:"required,email"`
+	Password            string `json:"password" binding:"required,min=12"`
+	PasswordConfirmation string `json:"password_confirmation" binding:"required,eqfield=Password"`
+}
+
+func isValidPassword(password string) error {
```

golangci-lint (1.64.8)

func `isValidPassword` is unused

(unused)